tsconfig.json

```
@@ -13,7 +13,11 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "baseUrl": ".",
+    "paths": {
+      "@components": ["./components/index.tsx"]
```

to follow what we do in other repos:
```suggestion
      "@/*": ["./*"]
```